pkgs/applications/networking/feedreaders/russ/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, stdenv
+, xorg
+, darwin
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "russ";
+  version = "2022-11-19-unstable";
```

Comment:
Apologies - one other thing I missed before is that `unstable`  should come before the date: "If a package is not a release but a commit from a repository, then the version attribute must be the date of that (fetched) commit. The date must be in "unstable-YYYY-MM-DD" format." (https://nixos.org/manual/nixpkgs/unstable/#sec-package-naming)

Reply:
derp. Fixed.